pkg/logger/logger.go

```diff
@@ -15,7 +15,21 @@ var (
 	once   sync.Once
 )
 
-func GetLogger() *zap.Logger {
+// Logger is wrapper for zap logger with module, it is singleton.
+type Logger struct {
+	module string
+	log    *zap.Logger
+}
+
+// GetLogger return logger with module name
+func GetLogger(module string) *Logger {
+	return &Logger{
+		module: module,
```

Comment:
The `module` is not used.

Reply:
format log msg using module name, list "[module name]:msg"